ansible/configs/sap-integration/files/fio-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```

Comment:
same, this was for another config. please go through and remove files that you are not using.

Reply:
done